chatkit-core/src/main/kotlin/com/pusher/chatkit/users/api/UserSubscriptionDispatcher.kt

```diff
@@ -10,20 +11,26 @@ import com.pusher.chatkit.state.RoomUpdated
 import org.reduxkotlin.Dispatcher
 
 internal class UserSubscriptionDispatcher(
-    private val joinedRoomsStateDiffer: JoinedRoomsStateDiffer,
     private val joinedRoomApiTypeMapper: JoinedRoomApiTypeMapper,
+    private val userApiTypeMapper: UserApiTypeMapper,
+    private val joinedRoomsStateDiffer: JoinedRoomsStateDiffer,
```

Comment:
```suggestion
    private val userApiTypeMapper: UserApiTypeMapper,
    private val joinedRoomsStateDiffer: JoinedRoomsStateDiffer,
    private val joinedRoomApiTypeMapper: JoinedRoomApiTypeMapper,
```
more logical order matching the order in which the dependencies are used

Reply:
Disagree - group the mappers together because they perform similar tasks.

Reply:
Oh sorry I missed out your feedback to realise that the code has been changed due to it. Such grouping here also represents a logical order and visually looks neater so I'm fine with it :) Also there's no point of giving it too much thought here as the differ here will be gone, to then reemerge like phoenix from the ashes and become the Late™ Differ 🤩

Just `userApiTypeMapper` first please as that's what we need first (already at the connection state) so it's more logical to put it first.

Reply:
Hmm. Have put the `userApiTypeMapper` first but kept the mappers together.